milli/src/update/settings.rs

```diff
@@ -1223,16 +1249,19 @@ impl InnerIndexSettings {
             embedding_configs,
             existing_fields,
             geo_fields_ids,
+            non_searchable_fields_ids: vectors_fids.clone(),
+            non_faceted_fields_ids: vectors_fids.clone(),
```

Comment:
I don't really get why you need *both* `non_faceted_fields_ids` and `faceted_fields_ids`, can you expand on this design? (maybe as comment where the `non_*_fields_ids` are defined)

Reply:
technically for this PR I don’t need it, but I would like to do a second PR where we also remove the `_geo` fields from the searchable (but not from the faceted).

I didn’t do it here since the PR is already quite big